hack/ginkgo-e2e.sh

```diff
@@ -152,7 +155,18 @@ CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-${KUBE_CONTAINER_RUNTIME:-}}
 # Add path for things like running kubectl binary.
 PATH=$(dirname "${e2e_test}"):"${PATH}"
 export PATH
-"${ginkgo}" "${ginkgo_args[@]:+${ginkgo_args[@]}}" "${e2e_test}" -- \
+
+# If the program is invoked with E2E_DEBUG=y we use delve instead.
+# NOTE: for this to work the e2e.test binary has to be compiled with
```

Comment:
this comment really ought to be on the env "option" above / early in the script.

Reply:
I moved this above